cmd/kube-apiserver/app/options/options.go

```diff
@@ -144,6 +146,11 @@ func (s *ServerRunOptions) Flags() (fss cliflag.NamedFlagSets) {
 	s.EgressSelector.AddFlags(fss.FlagSet("egress selector"))
 	s.Admission.AddFlags(fss.FlagSet("admission"))
 
+	// TODO(RainbowMango): move it to genericoptions before next flag comes.
+	mfs := fss.FlagSet("metrics")
+	mfs.StringVar(&s.ShowHiddenMetrics, "show-hidden-metrics", s.ShowHiddenMetrics,
```

Comment:
`--show-hidden-metrics-for-version` would be slightly clearer; up to you, though?

Reply:
That sounds reasonable to me.

Reply:
+1

Reply:
Done.